src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -126,55 +110,35 @@ $overlayShapeZIndex: 1;
   }
 }
 
-.HeroRecommendation-heading,
-.HeroRecommendation-body,
-.HeroRecommendation-link {
-  position: relative;
-  z-index: $overlayShapeZIndex + 1;
-}
-
-.HeroRecommendation-purple {
-  $heroPurple1: #20123a;
-  $heroPurple2: #712290;
-  $heroPurple3: #592acb;
-  $heroPurple4: #312a65;
-  $heroPurple5: #9059ff;
-  $heroPurple6: #e31587;
-  $heroPurple7: #312a65;
-
-  background-image: linear-gradient($heroPurple1, $heroPurple2);
-
-  .HeroRecommendation-gradientA-startColor {
-    stop-color: $heroPurple3;
-  }
-
-  .HeroRecommendation-gradientA-endColor {
-    stop-color: $heroPurple4;
-  }
-
-  .HeroRecommendation-gradientB-startColor {
-    stop-color: $heroPurple5;
-  }
-
-  .HeroRecommendation-gradientB-endColor {
-    stop-color: $heroPurple6;
-  }
-
-  .HeroRecommendation-solidSwoosh {
-    fill: $heroPurple7;
-  }
+@mixin gradientStyle($stopColor) {
+  background-image: linear-gradient($color-ink-80, $stopColor);
 
   .HeroRecommendation-link {
     &,
     &:hover,
     &:active {
-      // Fill the background on small screens in case the overlayShape
-      // overlaps the button. Also, fill the background when hovering.
-      background-color: $heroPurple2;
-    }
-
-    @include respond-to(large) {
-      background-color: transparent;
+      background-color: $stopColor;
     }
   }
 }
+
+// Styles that correspond to gradients returned by the hero API
+.color-ink-80-color-blue-70 {
+  @include gradientStyle($color-blue-70);
```

Comment:
I did do this because, as of right now, the start color is always the same. But we did design the API to be able to return both a start and end color, to cover the case if the start color ever changes, so it would be nice to bake in support for that here.

IOW, I will update the mixin to accept the start color.